static/app/components/highlightModalContainer.tsx

```diff
@@ -0,0 +1,46 @@
+import {Fragment} from 'react';
+import styled from '@emotion/styled';
+
+import BottomLeft from 'sentry-images/pattern/highlight-bottom-left.svg';
+import TopRight from 'sentry-images/pattern/highlight-top-right.svg';
+
+type Props = {
+  topWidth: string;
+  bottomWidth: string;
+  children: React.ReactNode;
+};
+
+export default function HighlightModalContainer({
```

Comment:
@roggenkemper when this PR is merged, could you make a PR in Getsentry to use this new `HighlightModalContainer` instead of the one in Getsentry?

Reply:
yup!